src/handlers/metagraphHandler.ts

```diff
@@ -363,9 +363,9 @@ export const currencyTransactionsByDestination = async (
   }
 };
 
-const balanceOrZeroFn = async (balance, address) => {
+const balanceOrZeroFn = async (balance, address, ordinal) => {
   if (balance === null) {
-    const snapshot_ordinal = (await latestMetagraphSnapshot())?.ordinal;
+    const snapshot_ordinal = isFinite(ordinal)? ordinal:(await latestMetagraphSnapshot())?.ordinal;
```

Comment:
Nitpick 
```suggestion
    const snapshot_ordinal = isFinite(ordinal)? ordinal : (await latestMetagraphSnapshot())?.ordinal;
```

Reply:
I like it, but prettier changed it to something different :P